easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.1.1q-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'OpenSSL'
+version = '1.1.1q'
+
+homepage = 'https://www.openssl.org/'
+description = """The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
+ and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1)
+ protocols as well as a full-strength general purpose cryptography library. """
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.openssl.org/source/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['d7939ce614029cdff0b6c20f0e2e5703158a489a72b2507b8bd51bf8c8fd10ca']
+
+dependencies = [('zlib', '1.2.11')]
+
+builddependencies = [
+    ('Perl', '5.32.1'),
```

Comment:
I changed it to minimal, but it seems that the rest are all build with the full perl...

Reply:
>it seems that the rest are all build with the full perl.

They are, but one of my fellow maintainers sent me a chat that that might be overkill. It's not a huge issue, but it's nice to use the `-minimal`  version of `Perl` as a dependency since it's a bit smaller/faster to build.

Reply:
Even better. Less overhead, the better :+1: